HLTrigger/HLTanalyzers/src/HLTJets.cc

```diff
@@ -587,19 +588,19 @@ void HLTJets::analyze(edm::Event const& iEvent,
  if( caloTowersCleanerUpperR45.isValid() ){
    towersUpperValid = true;
    for( CaloTowerCollection::const_iterator tow = caloTowersCleanerUpperR45->begin(); tow != caloTowersCleanerUpperR45->end(); tow++){
-     towersUpper.insert(tow->id().denseIndex());
+     towersUpper.insert(cttopo->denseIndex(tow->id()));
```

Comment:
@kpedro88 - at this time the pointer to CaloTowerTopology is not defined. The analyser will crash.